omnigibson/object_states/draped.py

```diff
@@ -0,0 +1,56 @@
+from omnigibson.object_states.kinematics import KinematicsMixin
+from omnigibson.object_states.object_state_base import BooleanState, RelativeObjectState
+from omnigibson.object_states.touching import Touching
+from omnigibson.utils.constants import PrimType
+from omnigibson.utils.object_state_utils import sample_cloth_on_rigid
+import omnigibson.utils.transform_utils as T
+from omnigibson.macros import create_module_macros
+
+import omnigibson as og
+
+from scipy.spatial import ConvexHull, HalfspaceIntersection
+from scipy.spatial.qhull import QhullError
+import numpy as np
+import trimesh
+import itertools
+
+# Create settings for this module
+m = create_module_macros(module_path=__file__)
+
+# Maximum velocity allowed for the draped object
+m.DRAPED_MAX_VELOCITY = 0.05
+
+class Draped(KinematicsMixin, RelativeObjectState, BooleanState):
+    @staticmethod
+    def get_dependencies():
+        return KinematicsMixin.get_dependencies() + RelativeObjectState.get_dependencies() + [Touching]
```

Comment:
nit: in general, might be safer to make this more programmatic -- e.g.: call super().get_dependencies() first and then add to that

Reply:
I don't think I understand this fully. Let's do a proper cleanup in the next PR.